docs/publish/centralized-deployment.md

```diff
@@ -13,7 +13,7 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
 The Office 365 admin center currently supports the following scenarios:
 
 - Centralized Deployment of new and updated add-ins to individuals, groups, or an organization.
-- Deployment to multiple platforms, including Windows, Mac, iOS, Android, and on the web.
+- Deployment to multiple platforms, including Windows, Mac and on the web. For Outlook, Deployment to iOS and Android is also supported 
```

Comment:
```suggestion
- Deployment to multiple client platforms, including Windows, Mac, the web, and  iPad. For Outlook, deployment to iOS and Android is also supported.
```

Reply:
@ElizabethSamuel-MSFT : 

Support on iPad is Not present for Centralized Deployment.

For Outlook, we only want to call out support on Mobile specifically. Mentioning about iOS, is causing users to assume that iPad is supported as well, but that is not the case. 
CD Support is present only for Outlook on Mobile iOS and Android

While we do not have add-in story for Word, Excel and PowerPoint on Mobile (iOS and Android)

Reply:
Actually, we do have an add-in story for Word, Excel, and PowerPoint on iPad (see the [availability page](https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability))...